bokehjs/src/lib/models/widgets/autocomplete_input.ts

```diff
@@ -180,6 +190,7 @@ export class AutocompleteInput extends TextInput {
     this.define<AutocompleteInput.Props>({
       completions:    [ p.Array, [] ],
       min_characters: [ p.Int,   2  ],
+      case_sensitive: [ p.Boolean,  false ],
```

Comment:
Case sensitive was the original behavior, so to provide backwards compatibility, the default must be `true`. We can change that in bokeh 3.0 if we so desire.

Reply:
Good point. Got it, made the change.